deployment/terraform/create.go

```diff
@@ -179,6 +179,12 @@ func (t *Terraform) setupAppServers(output *Output, extAgent *ssh.ExtAgent, uplo
 			// Upload binary if needed.
 			if uploadBinary {
 				mlog.Info("Uploading binary", mlog.String("host", ip))
+				cmd := "sudo service mattermost stop || true"
```

Comment:
Why do we need the `|| true`? I thought the stop command would succeed whether or not the service is running.

Reply:
It will fail the first time, because the service isn't even installed.

Reply:
The service file has been uploaded at that point. Maybe we just need a `systemctl daemon-reload`?

Reply:
Yes, should be fine.